### ISSUE TITLE
[APP] ICS calendar access (create/edit/delete event)

### ISSUE THREAD

**Issue by asheroto** [PipedreamHQ/pipedream#4332]
**Name of app / service**
ICS calendar files

**Link to developer documentation**
https://www.lifewire.com/ics-file-2622749
https://docs.fileformat.com/email/ics/

**Is lack of support preventing you from building workflows, or do you have a workaround?**
Yes, I'm stuck for now!

**Are there specific actions, or triggers, you'd like to see for this app? Please let us know here or use the Action and Trigger issue templates to open requests for each!**
- Ability to create/edit/delete calendars

ICS files are common with Outlook, Gmail, and a few others programs. My goal is to have a calendar entry created on my calendar when a certain action takes place. The ics file links are generally able to support editing rights, too. You usually set the permissions in your client and then share an ICS url,

If you have a Google account, you can visit https://calendar.google.com and click the Settings cog at the top right, select a calendar on the left, and go to Import/Export. The ICS URL is at the very bottom.

**Comment by dannyroosevelt** [comment]
@asheroto I don't think there's a web API for Pipedream to interact with ICS files, right?

Have you seen this done elsewhere in other platforms?

**Comment by asheroto** [comment]
I haven't seen around the net as much as I used to. For a time it was the only option to work with calendars. Nowadays so many use Gmail (Google) or Office 365 (Microsoft) and they have their own APIs, which Pipedream does support, of course.  I did put in a request for Exchange Server integration.

ICS / iCal format may not be in big demand, not sure - it's just a long-standing calendar standard.

[RFC 5545](https://www.rfc-editor.org/rfc/rfc5545) was written in 2009, so that gives away its age.
